### DISCUSSION TITLE
Image Resize Algorithm

### DISCUSSION THREAD

**Discussion by mgth** [AvaloniaUI/Avalonia#14327] [Q&A]
I could not find the way in image control to define which algorithm to use for resizing. I need the simplest one with no blur at all.

**Comment by thevortexcloud** [comment]
`Bitmap.DecodeTo` just wraps a bunch of Skia calls. You can do pretty much the same thing yourself or something completely different. EG this is how I do it: https://github.com/AvaloniaUI/Avalonia/discussions/13239#discussioncomment-7809111

**Comment by stevemonaco** [comment]
> I could not find the way in image control to define which algorithm to use for resizing. I need the simplest one with no blur at all.

The Image control shouldn't resize images, only rescale. If you mean rescale, then try the following:
```xml
<Image Source="Assets/someimage.png" RenderOptions.BitmapInterpolationMode="None" />
```
